libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -2516,7 +2516,7 @@
     <input name="scale" type="float" value="1.0"/>
     <input name="normal" type="vector3" defaultgeomprop="Nworld"/>
     <input name="tangent" type="vector3" defaultgeomprop="Tworld"/>
-    <output name="out" type="vector3" default="0.0, 1.0, 0.0"/>
+    <output name="out" type="vector3" defaultinput="0.0, 1.0, 0.0"/>
```

Comment:
If defaultinput is to be used instead of default, then it must name an input. In this case the appropriate input would be `normal`. So change to `defaultinput="normal"`.